routes/follow.go

```diff
@@ -107,13 +108,52 @@ func (r *followingHandler) Get(c *gin.Context) {
 	c.JSON(http.StatusOK, result)
 }
 
+func (r *followingHandler) GetByResource(c *gin.Context) {
+	input := struct {
+		Resource string   `json:"resource"`
+		Ids      []string `json:"ids"`
+	}{}
+	c.ShouldBindJSON(&input)
+
+	if len(input.Ids) == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"Error": "Bad Resource ID"})
+		return
+	}
+
+	switch input.Resource {
+	case "member":
```

Comment:
should we have indent here?

Reply:
The code was formatted by go fmt, it's ok for sure we can change the switch-case indentation, but I guess that following the default style is better for us.